vnext/Microsoft.ReactNative/IReactDispatcher.idl

```diff
@@ -30,6 +30,6 @@ namespace Microsoft.ReactNative {
     static IReactDispatcher UIThreadDispatcher{ get; };
 
     // Get name of the UIDispatcher property for the IReactPropertyBag.
-    static IReactPropertyName UIDispatcherProperty();
+    static IReactPropertyName UIDispatcherProperty { get; };
```

Comment:
>UIDispatcherProperty [](start = 30, length = 20)

just checking: Is it common in WinRT to post-fix properties? Or should it be `UIDispatcher` now that we added a getter?

Reply:
Note that this is not a real property that returns UIDispatcher. It is a property that can be used to access UIDispatcher from a property bag. This is why it has the 'Property' suffix.
E.g. `properties.Get(ReactDispatcherHelper.UIDispatcherProperty)`.

---
In reply to: [430588974](https://github.com/microsoft/react-native-windows/pull/5007#discussion_r430588974) [](ancestors = 430588974)